src/utils/date-time.ts

```diff
@@ -1,26 +1,15 @@
-export class Utils {
-  private static leadingZeroFormatter: Intl.NumberFormat =
-    new Intl.NumberFormat(undefined, {
-      minimumIntegerDigits: 2,
-    });
-
-  public static normalizeDuration(time: number) {
-    const seconds = Math.floor(time % 60);
-    const minutes = Math.floor(time / 60) % 60;
-    const hour = Math.floor(time / 3600);
-    if (isNaN(seconds)) return `00:00`;
+import { addSeconds, format } from "date-fns";
 
-    if (hour === 0) {
-      return `${minutes}:${this.leadingZeroFormatter.format(seconds)}`;
-    } else {
-      return `${hour}:${this.leadingZeroFormatter.format(
-        minutes,
-      )}:${this.leadingZeroFormatter.format(seconds)}`;
-    }
+export class UtilsDate {
+  public static normalizeDuration(seconds: number) {
+    try {
+      const helperDate = addSeconds(new Date(0), seconds);
+      return format(helperDate, "mm:ss");
+    } catch (e) {}
```

Comment:
You can use `console.error` here
```suggestion
    } catch (e) {
      console.error(e)
    }
```